src/06_tuples.py

```diff
@@ -35,10 +35,13 @@ def dist(a, b):
 # Write a function `print_tuple` that prints all the values in a tuple
 
 # YOUR CODE HERE
+def print_tuple(c):
+    for i in c:
+        print(i)
 
 t = (1, 2, 5, 7, 99)
 print_tuple(t)  # Prints 1 2 5 7 99, one per line
 
```

Comment:
Great work so far Humberto, they say Tuples could be difficult to tackle!  :star: